docs/requirements.txt

```diff
@@ -1,20 +1 @@
-black
-click
-importnb
-ipykernel
-IPython
-isort
-myst-parser
-nbconvert
-nbformat
-nbsphinx
-pre-commit
-pydata-sphinx-theme
-pytest
-sphinx_rtd_theme
-sphinx-autoapi
-sphinx-copybutton
-sphinx-copybutton
-sphinx-material
-sphinxcontrib-bibtex
-stringcase
+jupyter-book
```

Comment:
Hi @tonyfast, is this enough for building the docs? I noticed that it has all the sphinx-related dependencies that we might need, but is there anything else we need to add as well?

Reply:
jupyter-book has a ton of dependencies. most of them are to incorporate markdown/notebooks as sphinx documentation source. all those prior deps are taken care of by jupyter-book.